src/main/java/frc/robot/turret/simulation/SimulationConstants.java

```diff
@@ -1,9 +1,14 @@
 package frc.robot.turret.simulation;
 
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
 import edu.wpi.first.math.controller.PIDController;
 
 public class SimulationConstants {
```

Comment:
maybe make this class protected and the things in it also

Reply:
out of scope

Reply:
ill solve it in another PR